ocs_ci/ocs/scale_lib.py

```diff
@@ -861,7 +865,10 @@ def construct_pvc_creation_yaml_bulk_for_kube_job(no_of_pvc, access_mode, sc_nam
     pvc_dict_list = list()
     for i in range(no_of_pvc):
         pvc_name = helpers.create_unique_resource_name("test", "pvc")
-        size = f"{random.randrange(5, 105, 5)}Gi"
+        if pvc_size is not None:
+            size = pvc_size
+        else:
+            size = f"{random.randrange(5, 105, 5)}Gi"
         pvc_data = templating.load_yaml(constants.CSI_PVC_YAML)
```

Comment:
better be done outside the loop, so you will do it once and in each iteration of the loop just change the values you need.